src/software/jetson_nano/ansible/playbooks/deploy_nano.yml

```diff
@@ -88,13 +86,15 @@
           when: "'start' in actions"
           tags: always
 
-      # Rescue section only runs if any task in this block fails
-      rescue:
-        - name: Error Recovery - Ensure Announcement Services are Running
+        - name: Disable Announcements
           become: true
           become_method: sudo
           ansible.builtin.systemd:
-            name: wifi_announcement
+            name: "{{ service_name }}"
+            enabled: no
             masked: no
-            daemon_reload: yes
-            state: started
+            state: stopped
+          with_items: ["wifi_announcement", "ethernet_announcement"]
+          loop_control:
+            loop_var: service_name
+          tags: always
```

Comment:
Is this here to stay? Or is the idea that after sometime (when all robots have disabled announcements service) we will remove this part of the playbook? similarly for the setup_nano change

Reply:
after sometime we will remove this

Reply:
Sounds good. can you create a ticket for removing this in the future (probably under the robot software project)